keywording/text.xml

```diff
@@ -168,7 +172,7 @@ archs (there is at least one case of a <c>vim</c> script which only worked on
 </p>
 
 <p>
-Note that most (non-x86) archs expect you to be on the arch team and bugzilla
+Note that most (non-<c>amd64/x86</c>) archs expect you to be on the arch team and bugzilla
```

Comment:
```suggestion
Note that most (non-<c>amd64</c>/<c>x86</c>) archs expect you to be on the arch team and bugzilla
```

(And check the resulting line length please.)

Reply:
Thanks!